meson.build

```diff
@@ -232,6 +232,11 @@ subdir('pldmtool')
 
 subdir('configurations')
 
+if get_option('oem-ibm').enabled()
+  subdir('config')
+  subdir('oem/ibm/pfw-sms')
```

Comment:
would be good to name the folder as pfw-sms to pfw-sms-utils

Reply:
Okay